Adding the UNSA Fortunate, a MIL Type Ship for Mercs

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Description:  Part of the UNSA's vanguard fleet design, the Fortunate is an attack ship, equipped with frontal armor and guns in order to pierce through hulls. This design includes a boarding port, capable delivering soldiers to the enemy's busted hull in order to subdue and eliminate the threats in close quarters. 


## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
I've added this ship to be another merc combat capable ship, since the mercs need some love. 

It's a attack ship with a focus on boarding, relying on it's armor and thrusters to get it close enough to an enemy ship and punch holes into the hull with it's front facing 57mm dravon cannons, and ASM missiles. 

Once the hull is breached, or if the ship is close enough, the boarding party can fly close with jetpacks in order to eliminate the enemy crew in close quarters combat. 

The ship's crew would be equipped with UNSA Armored EVA suits, 5x Enforcer shotguns, 5x breaching hammers. 


## How to test
<!-- Describe the way it can be tested -->
load up the .yml file in an IDE, should work!

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="763" height="859" alt="fortunate ship live" src="https://github.com/user-attachments/assets/bbf3a37f-24ae-49af-a351-6cf980920aba" />
<img width="759" height="852" alt="fortunate ship" src="https://github.com/user-attachments/assets/bde2c57c-193b-4cb4-a1ea-b34b05d32b0f" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: UNSA Fortunate, a MIL ship made for boarding

Rubeebeebee committed:
adding the UNSA Fortuante, a MIL type ship for mercs

monolith8319 committed:
chore: automatically update REUSE headers

johnnny34y commented:
the UNSA is armed baby

Rubeebeebee committed:
updated the front walls to not be two diagonals and added a fax and holopad.

Rubeebeebee committed:
Merge branch 'Fortunate-branch' of https://github.com/Rubeebeebee/Monolith into Fortunate-branch

Rubeebeebee commented:
updated it so the front boarding bay has it's two diagonal walls reduced to just one, and added a fax and holopad. 
<img width="472" height="679" alt="fortunate tweaks2" src="https://github.com/user-attachments/assets/bdf27983-4189-4951-a44a-931f10fcbdd6" />
<img width="651" height="362" alt="fortunate tweaks1" src="https://github.com/user-attachments/assets/890f2851-82c7-4495-9e50-2677f83ae0b3" />

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Fortunate-branch

OneZerooo0 reviewed (changes requested):
Please reduce armory to a kammerer and melee weapon at most
No need for four mercenary EVA suit cabinets when you have four armoured EVAs in the front
Needs to be at least 150k for its armament

Rubeebeebee committed:
changed the price to 152k

Rubeebeebee committed:
removed the 4 merc EVA suits from the docks, and adjusted the armory to only contain 1x kammerer, and 1x breaching hammer

Rubeebeebee committed:
Merge branch 'Fortunate-branch' of https://github.com/Rubeebeebee/Monolith into Fortunate-branch

Rubeebeebee commented:
> Please reduce armory to a kammerer and melee weapon at most No need for four mercenary EVA suit cabinets when you have four armoured EVAs in the front Needs to be at least 150k for its armament

removed the 4 merc EVA suits from the docks, and adjusted the armory to only contain 1x kammerer, and 1x breaching hammer

made the price into 152k to suit it's weaponry

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
@OneZerooo0 THANK YOUUUU!!
